### ISSUE TITLE
`ghc-9.4.8`: `llvmize` broken on glibc

### ISSUE THREAD

**Issue by 0x6d6e647a** [gentoo-haskell/gentoo-haskell#1527]
Tried to upgrade to `dev-lang/ghc-9.4.8` and got the following error during source preparation:
```
find: ‘/var/tmp/portage/dev-lang/ghc-9.4.8/work/usr/bin’: No such file or directory                                    
sed: no input files
```
See attached build.log for more: [dev-lang_ghc-9.4.8_build.log.txt](https://github.com/gentoo-haskell/gentoo-haskell/files/14628637/dev-lang_ghc-9.4.8_build.log.txt)

**Comment by hololeap** [comment]
Could you also post the output of `emerge --info`?

**Comment by 0x6d6e647a** [comment]
[emerge_info.txt](https://github.com/gentoo-haskell/gentoo-haskell/files/14629156/emerge_info.txt)

**Comment by hololeap** [comment]
I think as a workaround, you can disable the `llvm` USE flag.

**Comment by 0x6d6e647a** [comment]
Removing the `llvm` USE flag got it building!

No big deal, LLVM is a new GHC feature so I'm not surprised there's some hiccups. Was just hoping to play with the new LLVM features. Thanks for your efforts @hololeap, I really appreciate it!

**Comment by 0x6d6e647a** [comment]
Quick update, GHC built fine with the USE flag removed.

**Comment by hololeap** [comment]
Good to hear. Thanks, it would still be good to keep this open so we can take a look at it later.

**Comment by hololeap** [comment]
`llvmize` is fixed now in 5b552b87259f189c1a7debfb97de4cc8960152c6 , but the build still fails with `USE=llvm` on glibc systems:

```
/usr/bin/ar: invalid option -- 'L'
```
([gist](https://gist.github.com/hololeap/f3d434525e1456d28223b4fdbeb51ff5#file-ghc-9-6-4-r1-glibc-llvm-log-L758))
